From 271d73604dd4dcdcf7babbdb071a1ee68218e3e5 Mon Sep 17 00:00:00 2001
From: Kamal Mostafa <kamal@canonical.com>
Date: Fri, 10 Aug 2012 14:06:10 -0700
Subject: [PATCH] UBUNTU: SAUCE: input: Cypress PS/2 Trackpad eliminate dead
 code

BugLink: http://launchpad.net/bugs/978807

Eliminiate the cytp_wakup() one-liner; we'll just call wake_up() directly
from psmouse-base.c instead.

Signed-off-by: Kamal Mostafa <kamal@canonical.com>
Signed-off-by: Tim Gardner <tim.gardner@canonical.com>
---
 drivers/input/mouse/cypress_ps2.c | 5 -----
 drivers/input/mouse/cypress_ps2.h | 1 -
 2 files changed, 6 deletions(-)

diff --git a/drivers/input/mouse/cypress_ps2.c b/drivers/input/mouse/cypress_ps2.c
index 41dab3e..8733676 100644
--- a/drivers/input/mouse/cypress_ps2.c
+++ b/drivers/input/mouse/cypress_ps2.c
@@ -119,11 +119,6 @@ static int cypress_ps2_ext_cmd(struct psmouse *psmouse, unsigned short cmd,
 	return rc;
 }
 
-inline void cytp_wakup(struct psmouse *psmouse)
-{
-	wake_up(&psmouse->ps2dev.wait);
-}
-
 static int cypress_ps2_read_cmd_status(struct psmouse *psmouse,
 				       unsigned char cmd,
 				       unsigned char *param)
diff --git a/drivers/input/mouse/cypress_ps2.h b/drivers/input/mouse/cypress_ps2.h
index dda6c00..de4aa58 100644
--- a/drivers/input/mouse/cypress_ps2.h
+++ b/drivers/input/mouse/cypress_ps2.h
@@ -203,6 +203,5 @@ struct cytp_data {
 int cypress_detect(struct psmouse *psmouse, bool set_properties);
 int cypress_init(struct psmouse *psmouse);
 bool cypress_supported(void);
-inline void cytp_wakup(struct psmouse *psmouse);
 
 #endif  /* _CYPRESS_PS2_H */
-- 
1.8.0

